docs/dynamic-plugins.md

```diff
@@ -361,6 +482,9 @@ By sequentially adding these plugins and allowing for a deployment restart after
 
 ### Using a custom NPM registry
 
+> [!WARNING]  
+> Using a custom NPM Registry and/or NPMJS is not recommended. Using [Container Registry](#consuming-dynamic-plugins-from-a-container-registry) or [Local Configuration](#local-configuration) is recommended and supported for dynamic plugins production environments.
```

Comment:
Maybe we should justify this first sentence why it's not recommended.  Basically that a dynamic plugin is a runtime artifact, and that it is not the best usage of an NPM registry to deliver runtime artifacts, especially in a container-based environment such as OCP.